binder/reconstruction-graphs.ipynb

```diff
@@ -0,0 +1,121 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from keyname import keyname as kn\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "import ptitprince as pt\n",
+    "import seaborn as sns\n",
+    "from teeplot import teeplot\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv(\"https://osf.io/ck47r/download\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "methodological_variables = [\n",
+    "#  'resolution',\n",
+    " 'subsampling-fraction',\n",
+    " 'trie-postprocess',  # pick one based on bias/accuracy\n",
+    "]\n",
+    "\n",
+    "evolutionary_variables = [\n",
+    " 'mut_distn',  # sensitivity analysis\n",
+    " 'num_generations',  # sensitivity analysis\n",
+    " 'num_islands',\n",
+    " 'num_niches',\n",
+    " 'p_island_migration',  # sensitivity analysis\n",
+    " 'p_niche_invasion',  # sensitivity analysis\n",
+    " 'population_size',  # doesn't change\n",
+    " 'tournament_size',\n",
+    "]\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = df[\n",
+    "    (df['subsampling-fraction'] == 1.0) &\n",
+    "    (df['trie-postprocess'] == 'naive')\n",
+    "]\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for group, group_df in df.groupby(methodological_variables + evolutionary_variables):\n",
+    "    teeplot.tee(\n",
```

Comment:
you could combine these two in an outer loop over `viz in sns.barplot, pt.RainCloud:`

Reply:
Done in 2d28d49